### PR TITLE
Missing SQL for the update process

### LINKED ISSUE #15048: AdminTabs controller can not be accessed - Menu BO Page
**Describe the bug**
`AdminTabs` admin controller doesn't appear in any menu. If you try to access directly by URL it returns an `Acces denied` error.

This functionality is not in any other controller.

**To Reproduce**
1. Open any AdminController
2. Change URL to `controller=AdminTabs`
3. Bypass token warning

**Additional information**
PrestaShop version: 1.7.6.0

### PR DESCRIPTION
**Pull request by ghost**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Access denied when acess localhost/myshop/admin123/index.php?controller=AdminTabs&toke=xxx with SuperAdmin And AdminTabs Tab don't exists<br><br>This pull request is will continue to my old pull request because  it's become unknown repository: https://github.com/PrestaShop/PrestaShop/pull/17501
| Type?         |  improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket ? | Fixes #15048 
| How to test?  | 1. Open any AdminController<br>2. Change URL to `controller=AdminTabs`<br>3. Bypass token warning

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19660)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: install-dev/upgrade/sql/1.7.7.0.sql [hunk 1]
```diff
@@ -800,3 +800,14 @@ VALUES (NULL, 'actionOrderMessageFormBuilderModifier', 'Modify order message ide
...
+INSERT IGNORE INTO `ps_authorization_role` (`slug`) VALUES
+  ('ROLE_MOD_TAB_DEFAULT_CREATE'),
+  ('ROLE_MOD_TAB_DEFAULT_READ'),
+  ('ROLE_MOD_TAB_DEFAULT_UPDATE'),
+  ('ROLE_MOD_TAB_DEFAULT_DELETE');
+
+INSERT IGNORE INTO `ps_access` (`id_profile`, `id_tab`, `view`, `add`, `edit`, `delete`) VALUES ('1', (SELECT `id_tab` FROM `ps_tab` t WHERE t.class_name = 'AdminTAbs' LIMIT 1), '1', '1', '1', '1');
+
+INSERT IGNORE INTO `ps_tab_lang` (`id_lang`, `id_tab`, `name`) VALUES
+	(1, (SELECT `id_tab` FROM `ps_tab` WHERE `class_name` = 'AdminTabs'), 'Menus');
```
Comment (PululuK) [inline install-dev/upgrade/sql/1.7.7.0.sql]: Use `PREFIX_` instead of `ps_`

**Review by PululuK — CHANGES_REQUESTED** [review]

**Review by PululuK — CHANGES_REQUESTED** [review]
The develop branch is now on 1.7.8.0, See https://github.com/PrestaShop/PrestaShop/blob/develop/app/AppKernel.php#L33

Please could you put your query here : https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/upgrade/sql/1.7.8.0.sql

File: install-dev/upgrade/sql/1.7.7.0.sql (hunk 1 above)
Comment (Progi1984) [reply install-dev/upgrade/sql/1.7.7.0.sql]: Well done ;)

Comment (PululuK) [reply install-dev/upgrade/sql/1.7.7.0.sql]: I'm getting older, but I still have my eyes 🤠

File: install-dev/upgrade/sql/1.7.7.0.sql [hunk 2]
```diff
@@ -800,3 +800,14 @@ VALUES (NULL, 'actionOrderMessageFormBuilderModifier', 'Modify order message ide
        (NULL, 'actionCreditSlipGridPresenterModifier', 'Modify credit slip grid template data',
         'This hook allows to modify data which is about to be used in template for credit slip grid', '1')
 ;
+
+INSERT IGNORE INTO `ps_authorization_role` (`slug`) VALUES
```
Comment (PierreRambaud) [inline install-dev/upgrade/sql/1.7.7.0.sql]:
```suggestion
INSERT IGNORE INTO `PREFIX_authorization_role` (`slug`) VALUES
```

File: install-dev/upgrade/sql/1.7.7.0.sql [hunk 3]
```diff
@@ -800,3 +800,14 @@ VALUES (NULL, 'actionOrderMessageFormBuilderModifier', 'Modify order message ide
...
+  ('ROLE_MOD_TAB_DEFAULT_CREATE'),
+  ('ROLE_MOD_TAB_DEFAULT_READ'),
+  ('ROLE_MOD_TAB_DEFAULT_UPDATE'),
+  ('ROLE_MOD_TAB_DEFAULT_DELETE');
+
+INSERT IGNORE INTO `ps_access` (`id_profile`, `id_tab`, `view`, `add`, `edit`, `delete`) VALUES ('1', (SELECT `id_tab` FROM `ps_tab` t WHERE t.class_name = 'AdminTAbs' LIMIT 1), '1', '1', '1', '1');
```
Comment (PierreRambaud) [inline install-dev/upgrade/sql/1.7.7.0.sql]:
```suggestion
INSERT IGNORE INTO `PREFIX_access` (`id_profile`, `id_tab`, `view`, `add`, `edit`, `delete`) VALUES ('1', (SELECT `id_tab` FROM `PREFIX_tab` t WHERE t.class_name = 'AdminTAbs' LIMIT 1), '1', '1', '1', '1');
```

File: install-dev/upgrade/sql/1.7.7.0.sql [hunk 4]
```diff
@@ -800,3 +800,14 @@ VALUES (NULL, 'actionOrderMessageFormBuilderModifier', 'Modify order message ide
...
+
+INSERT IGNORE INTO `ps_tab_lang` (`id_lang`, `id_tab`, `name`) VALUES
```
Comment (PierreRambaud) [inline install-dev/upgrade/sql/1.7.7.0.sql]:
```suggestion
INSERT IGNORE INTO `PREFIX_tab_lang` (`id_lang`, `id_tab`, `name`) VALUES
```

**Comment by HamzaElghanoui** [comment]
Thanks @PierreRambaud for your suggested but I can't accept it because I must change my codes from install-dev/upgrade/sql/1.7.7.0.sql  to install-dev/upgrade/sql/1.7.8.0.sql
Thank you

**Comment by HamzaElghanoui** [comment]
> The develop branch is now on 1.7.8.0, See https://github.com/PrestaShop/PrestaShop/blob/develop/app/AppKernel.php#L33
> 
> Please could you put your query here : https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/upgrade/sql/1.7.8.0.sql

@PululuK Done

File: install-dev/upgrade/sql/1.7.8.0.sql [hunk 5]
```diff
@@ -10,3 +10,14 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
...
   (NULL, 'actionModuleUninstallAfter', 'Module uninstall after', 'This hook is called at the end of module uninstall process', '1')
 ;
+
+INSERT IGNORE INTO `PREFIX_authorization_role` (`slug`) VALUES
+  ('ROLE_MOD_TAB_DEFAULT_CREATE'),
+  ('ROLE_MOD_TAB_DEFAULT_READ'),
+  ('ROLE_MOD_TAB_DEFAULT_UPDATE'),
+  ('ROLE_MOD_TAB_DEFAULT_DELETE');
+
+INSERT IGNORE INTO `PREFIX_access` (`id_profile`, `id_tab`, `view`, `add`, `edit`, `delete`) VALUES ('1', (SELECT `id_tab` FROM `PREFIX_tab` t WHERE t.class_name = 'AdminTAbs' LIMIT 1), '1', '1', '1', '1');
```
Comment (Progi1984) [inline install-dev/upgrade/sql/1.7.8.0.sql]:
```suggestion
INSERT IGNORE INTO `PREFIX_access` (`id_profile`, `id_tab`, `view`, `add`, `edit`, `delete`) VALUES ('1', (SELECT `id_tab` FROM `PREFIX_tab` t WHERE t.class_name = 'AdminTabs' LIMIT 1), '1', '1', '1', '1');
```

**Review by Progi1984 — COMMENTED** [review]
A small suggestion ;) And it's ok for me

**Comment by HamzaElghanoui** [comment]
> A small suggestion ;) And it's ok for me

@Progi1984 Good observation

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
Thanks @HamzaElghanoui !

**Pull request merged**